.github/workflows/tests.yml

```diff
@@ -145,6 +145,12 @@ jobs:
                     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
                     SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
+            -   name: Upload backend source maps
```

Comment:
Why do this here and not in the deployment workflow? It might lead to undesirable side effects, where PRs change the source map but the deployed version is different, no?